### ISSUE TITLE
Filler's issue

### ISSUE THREAD

**Issue by sfchipan** [BuildCraft/BuildCraft#2796]
The Stairs pattern(up) don't build from the lowest.
![2015-06-13_14 01 42](https://cloud.githubusercontent.com/assets/4555814/8145062/4b7fc776-122b-11e5-909e-a8f35adc0dfe.png)
The Flatten pattern just fill one level.
![2015-06-14_00 08 23](https://cloud.githubusercontent.com/assets/4555814/8145065/58fee634-122b-11e5-99c3-de70a93c656d.png)
Thanks for reply.

**Comment by asiekierka** [comment]
The Flatten pattern only fills one level, don't you mean Horizon?

About Stairs, fixed.

**Comment by sfchipan** [comment]
Yes, I mean Flatten. I remember the Flatten can fill all the air blocks below the selected area, not just one level in 1.6.4

**Comment by sfchipan** [comment]
Mm...like this:
![2015-06-14_21 36 25](https://cloud.githubusercontent.com/assets/4555814/8148814/20e0ea62-12e0-11e5-840e-136e35ea269d.jpg)

**Comment by AEnterprise** [comment]
outdated milestone, is this still an issue?

**Comment by AlexIIL** [comment]
Yea. But its odd: the filler will build stairs one up from where they should be if the sides are of different lengths. There are probably more edge cases like this though.

**Comment by AEnterprise** [comment]
no longer an issue with the new filler patterns

**Comment by AlexIIL** [comment]
Now an issue again :P

**Comment by AEnterprise** [comment]
is this still relevant with all the changes already made to the filler?
